static/js/upload_widget.js

```diff
@@ -21,6 +21,8 @@ exports.build_widget = function (
     input_error, // jQuery object for error text
     clear_button, // jQuery button to clear last upload choice
     upload_button, // jQuery button to open file dialog
+    emoji_preview = null,
+    emoji_image,
```

Comment:
Let's keep these names too consistent with what used in the `build_emoji_upload_widget` method.